### ISSUE TITLE
suppress 'rsession: no process found' messages when stopping RStudio Server

### ISSUE THREAD

**Issue by kevinushey** [rstudio/rstudio#3269]
### System details

    RStudio Edition : Server
    RStudio Version : v1.1.453
    OS Version      : Ubuntu 18.04 64bit
    R Version       : NA

### Steps to reproduce the problem

While no R sessions are running, execute `sudo rstudio-server stop`.

### Describe the problem in detail

The message 'rsession: no process found' is emitted.

### Describe the behavior you expected

The message is misleading, and might lead a user to conclude something went wrong while attempting to shut down the server. We should suppress this message.

**Comment by kevinushey** [comment]
See e.g.

https://github.com/rstudio/rstudio/blob/68d30dbbaada445f55519c5ccf00dd934b825b11/src/cpp/server/extras/admin/rstudio-server.in#L39

I'm not sure why we suppress the `killall` messages for other processes but not `rsession`. We also seem to do this inconsistently (e.g. in `offline` we don't suppress for the kill of `rworkspaces`)

**Comment by kevinushey** [comment]
Similar 'noise' occurs when reinstalling RStudio Server:

```R
kevin@ubuntu:~/Downloads
$ sudo dpkg -i rstudio-server-1.2.1053-amd64.deb
[sudo] password for kevin:
(Reading database ... 203374 files and directories currently installed.)
Preparing to unpack rstudio-server-1.2.1053-amd64.deb ...
Unpacking rstudio-server (1.2.1053) over (1.2.1013) ...
Setting up rstudio-server (1.2.1053) ...
useradd: user 'rstudio-server' already exists
groupadd: group 'rstudio-server' already exists
rsession: no process found
Created symlink /etc/systemd/system/multi-user.target.wants/rstudio-server.service → /etc/systemd/system/rstudio-server.service.
● rstudio-server.service - RStudio Server
   Loaded: loaded (/etc/systemd/system/rstudio-server.service; enabled; vendor preset: enabled)
   Active: active (running) since Mon 2018-10-15 09:57:15 PDT; 1s ago
  Process: 10793 ExecStart=/usr/lib/rstudio-server/bin/rserver (code=exited, status=0/SUCCESS)
 Main PID: 10794 (rserver)
    Tasks: 3 (limit: 4915)
   CGroup: /system.slice/rstudio-server.service
           └─10794 /usr/lib/rstudio-server/bin/rserver

Oct 15 09:57:15 ubuntu systemd[1]: Starting RStudio Server...
Oct 15 09:57:15 ubuntu systemd[1]: Started RStudio Server.
```

It would be nice to avoid / suppress the output of:

```
useradd: user 'rstudio-server' already exists
groupadd: group 'rstudio-server' already exists
rsession: no process found
```

**Comment by asherkhb** [comment]
Also, this returns exit code 1, which causes general annoyance such as causing scripted versions (with set -o errexit) to fail,

```
[user@ip ~]$ sudo rstudio-server status
● rstudio-server.service - RStudio Server
   Loaded: loaded (/etc/systemd/system/rstudio-server.service; enabled; vendor preset: disabled)
   Active: active (running) since Fri 2019-05-24 16:48:38 MDT; 2min 59s ago
[...]
[user@ip ~]$ sudo rstudio-server stop
rsession: no process found
[user@ip ~]$ echo $?
1
[user@ip ~]$ sudo rstudio-server status
● rstudio-server.service - RStudio Server
   Loaded: loaded (/etc/systemd/system/rstudio-server.service; enabled; vendor preset: disabled)
   Active: inactive (dead) since Fri 2019-05-24 16:51:39 MDT; 7s ago
[...]
```

**Comment by jjesusfilho** [comment]
I am having the same issue.

**Comment by asherkhb** [comment]
should be fixed by #4857!
